spec/system/consumer/checkout/summary_spec.rb

```diff
@@ -35,6 +35,11 @@
     create(:shipping_method, require_ship_address: true,
                              name: "A Free Shipping with required address")
   }
+  let!(:payment_with_fee) {
+    create(:payment_method, distributors: [distributor],
+                            name: "Payment with Fee", description: "Payment with fee",
+                            calculator: Calculator::FlatRate.new(preferred_amount: 1.23))
+  }
```

Comment:
Was this missing before ?

Reply:
Yes, this is missing in master. We're currently not checking payment fees appear on the final step - we probably should...